tests/test_task.py

```diff
@@ -139,16 +139,16 @@ def setUp(self):
         # the rest result
         self.workspace_dir = tempfile.mkdtemp()
 
+    @retrying.retry(
+        stop_max_attempt_number=N_TEARDOWN_RETRIES,
+        wait_exponential_multiplier=250, wait_exponential_max=MAX_TRY_WAIT_MS)

```

Comment:
This is a cool library!

Reply:
Yes! I use it all the time. Especially useful when you're doing some kind of fetch over the internet that could fail.